src/edu/ucsd/library/xdre/web/MarcModsImportController.java

```diff
@@ -97,4 +106,17 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 		}
 		return new ModelAndView("marcModsImport", "model", dataMap);
 	}
+	
+	public static List<String> getCountryCodes (HttpServletRequest request) throws DocumentException {
```

Comment:
This does the job, but presents the user a long list of unsorted codes.  Please make this a code => name map, so the user can choose from the country names.